app/views/green_lanes/exemptions/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<h2>
+  Manage Green Lanes Exemptions
+</h2>
+
+<%= link_to 'Add a Exemption', new_green_lanes_exemption_path, class: 'govuk-button' %>
+
+<% if @exemptions.any? %>
+  <table>
+    <thead>
+    <tr>
+      <th>ID</th>
```

Comment:
Not sure we need to expose the exemptions id to the admin user - is it relevant? They will be thinking in terms of the code which is also unique?

Reply:
Removed the id